model/pdata/common.go

```diff
@@ -116,6 +120,13 @@ func NewAttributeValueArray() AttributeValue {
 	return AttributeValue{orig: &otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_ArrayValue{ArrayValue: &otlpcommon.ArrayValue{}}}}
 }
 
+// NewAttributeValueBytes creates a new AttributeValue with the given []byte value.
+// The caller must ensure the []byte passed in is not modified after the call is made, sharing the data
+// across multiple attributes is discouraged.
```

Comment:
is forbidden :) probably. We need to have a stronger comment

Reply:
👍